lm_eval/tasks/__init__.py

```diff
@@ -253,12 +253,13 @@ def _load_individual_task_or_group(
         name_or_config: Optional[Union[str, dict]] = None,
         parent_name: Optional[str] = None,
         update_config: Optional[dict] = None,
+        yaml_path: Optional[str] = None,
```

Comment:
TODO: check if this is the desired merge commit end result

Reply:
@lintangsutawika is going to make this change in a separate PR. I've reverted the change in this PR.